feconf.py

```diff
@@ -1449,3 +1449,6 @@ def get_empty_ratings():
     SUGGESTION_TYPE_TRANSLATE_CONTENT,
     SUGGESTION_TYPE_ADD_QUESTION
 ]
+
+# Prefix for all acl validation handlers.
+ACCESS_VALIDATORS_PREFIX = '/access_validator'
```

Comment:
ACCESS_VALIDATION_HANDLER_PREFIX perhaps -- otherwise, this starts with a verb and it refers to a "validator" which isn't the same thing as a "validation handler".

Try to also explain in the comment how this is used (e.g. what the naming scheme is for the different handlers which use it, with examples).

Reply:
done